src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -44,14 +44,23 @@
      */
     setColumns: function(component, dataColumns) {
         var columns = [];
-        columns.push({label: 'Donor', fieldName: 'donorLink', type: 'url', editable: false, typeAttributes: {label: {fieldName: 'donorName'}}});
+        columns.push({
+            label: 'Donor',
+            fieldName: 'donorLink',
+            type: 'url',
+            editable: false,
+            typeAttributes: {label: {fieldName: 'donorName'}}});
 
         dataColumns.forEach(function(col){
-            columns.push({label: col.label,
-                fieldName: col.fieldName,
-                type: col.type,
-                editable: !col.readOnly,
-                typeAttributes: JSON.parse(col.typeAttributes)});
+            if (col.type !== 'reference') {
```

Comment:
I'm missing the nuance of why aren't we including reference fields in the columns ... since I remember seeing them there! 

Unless it's because of the comment below for setDataImportFields() where there's a distinction between the read-only fields and editable fields in the grid?

Reply:
This could be a good thing to discuss in a group code review... but your assumption is correct! See the comment below on 89 for clarity on that.

Reply:
There are two separate "Column" objects -- one for the actual lookup in the form and one for the link output in the grid, since there is no native handling for reference datatype in datatable and it just displays as a raw id. We create our own separate "link" url-type Column and display that one in the datatable instead.

Reply:
It's not totally intuitive to think of the fields in the form as "columns" but I think I can let it go